packages/react/src/lib/felaExpandCssShorthandsPlugin.ts

```diff
@@ -0,0 +1,33 @@
+import * as _ from 'lodash'
+import * as _expand from 'css-shorthand-expand'
+
+// `css-shorthand-expand` is a CJS library, there are known issues with them:
+// https://github.com/rollup/rollup/issues/1267#issuecomment-446681320
+const expand = (_expand as any).default || _expand
+
+export default () => {
+  const expandCssShorthands = (styles: Object) => {
+    return _.keys(styles).reduce((acc, cssPropertyName) => {
+      const cssPropertyValue = styles[cssPropertyName]
+
+      if (typeof cssPropertyValue === 'object') {
+        return { ...acc, [cssPropertyName]: expandCssShorthands(cssPropertyValue) }
+      }
+
+      const expandedProps = expand(_.kebabCase(cssPropertyName), String(cssPropertyValue))
+      if (expandedProps) {
+        return { ...acc, ...convertKeysToCamelCase(expandedProps) }
+      }
+
+      return { ...acc, [cssPropertyName]: cssPropertyValue }
```

Comment:
here we may simplify and use ternary expression:

```js
return { 
  ...acc,  
  ...(expandedProps ? convertKeysToCamelCase(expandedProps) : { [cssPropertyName]: cssPropertyValue })
}
```

Reply:
There are multiple conditions there now, so I wouldn't complicate the return statements